ConwaysGameOfLife/GameController.cs

```diff
@@ -89,18 +89,25 @@ public string CreateValidUniverseString()
             return input;
         }
 
-        public string CreateValidLocationString(int gridWidth, int gridLength)
+        public string CreateValidLocationString()
         {
             var input = _input.ReadLine();
-            var isValidLocation = Validator.IsValidLocation(input, gridWidth, gridLength);
+            var isValidLocation = Validator.IsValidLocation(input, _universe.GridWidth, _universe.GridLength);
             while (!isValidLocation) 
             {
                 _output.WriteLine(Messages.InvalidInput);
                 _output.WriteLine($"{Messages.RequestLiveCell}.");
                 input = _input.ReadLine();
-                isValidLocation = Validator.IsValidLocation(input, gridWidth, gridLength);
+                isValidLocation = Validator.IsValidLocation(input, _universe.GridWidth, _universe.GridLength);
             }
             return input;
         }
+
+        public Cell SetLiveCellLocation(Location location)
+        {
+            var cell = _universe.GetCellAtLocation(location.X, location.Y); 
+            cell = new Cell(CellState.Alive, location.X, location.Y);
+            return cell;

```

Comment:
should this be returning the `cell`? or can we test the `Universe` in which the cell resides in another way?